src/pages/ColumnList/Pagination.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from "@emotion/styled";
+import { useState } from "react";
+
+import { useQueryParam } from "@/utils/hooks/useQueryParam";
+
+// eslint-disable-next-line react-refresh/only-export-components
+export const queryKey = "pageToken";
+
+type Props = {
+  totalResults: number;
+  resultsPerPage: number;
+};
+
+/**
+ * 컬럼, 상품 리스트에 담길 페이지네이션
+ *
+ * @params totalResults 전체 데이터 수
+ * @params resultsPerPage 페이지당 결과 수
+ * @returns
+ */
+export default function Pagination({ totalResults, resultsPerPage }: Props) {
+  const maxPageNum = Math.ceil(totalResults / resultsPerPage);
+
+  const { activeState, changeState } = useQueryParam(
+    queryKey,
+    [...Array(maxPageNum).keys()].map((i) => String(i + 1)),
+  );
+  const [pageNum, setPageNum] = useState(1);
+
+  const pageBtns = [
+    ...Array(maxPageNum === pageNum ? totalResults % resultsPerPage : resultsPerPage).keys(),
+  ].map((i) => resultsPerPage * (pageNum - 1) + i + 1);
+
+  const changePage = (page: number) => {
+    setPageNum(page);
+    changeState(String(resultsPerPage * (page - 1) + 1));
+  };
+
+  return (
+    <Container>
+      <PageBtn onClick={() => changePage(pageNum - 1)} disabled={pageNum === 1}>
```

Comment:
```suggestion
      <PageBtn aria-label="이전 페이지" onClick={() => changePage(pageNum - 1)} disabled={pageNum === 1}>
```

버튼 children으로 유저에게 보여주는 안내가 충분하지 않다면 요렇게 보충해볼 수도 있습니다. 코드 작성시 접근성도 챙겨보면 좋겠습니다.

Reply:
[feat: 웹 접근성 고려](https://github.com/kakao-tech-campus-2nd-step3/Team2_FE/pull/36/commits/370aaef422e70b2d756485b095129f8db37fc648)
한번도 생각해본적이 없는 문제네요..! 웹 접근성 관련해서 공부해보아야겠습니다. 감사합니다!